Documentation/ColumnsConfig/Type/Flex/Examples.rst

```diff
@@ -32,39 +32,24 @@ Notice the :xml:`<input_1>` tag:
 It's clear that the contents of :xml:`<input_1>` is a direct reflection of
 the field configurations we normally set up in the :php:`$GLOBALS['TCA']` array.
 
+..  _columns-flex-example-plugin:
 
 FlexForm in a plugin
 ====================
 
-The Data Structure for a FlexForm can also be loaded in the :sql:`pi_flexform`
+The data structure for a FlexForm can also be loaded in the :sql:`pi_flexform`
 field of the :sql:`tt_content` table by adding the following in the
-TCA Overrides of an extension:
+TCA Overrides of an extension, see this example from the EXT:blog_example:
 
-..  code-block:: php
-    :caption: EXT:my_extension/Configuration/TCA/Overrides/tt_content.php
+..  literalinclude:: _CodeSnippets/_tt_content_plugin.php
```

Comment:
Please activate linenumbers. IMO: lineos
Else it's hard to work with 18ff and 25ff below